pages/reference/base-images/base-images.md

```diff
@@ -112,18 +112,18 @@ Currently balenalib supports the following OS distributions and Language stacks,
 
 | Distribution | Default                | Supported Architectures                      |
 |---------|------------------------------|----------------------------------------------|
-| Debian  | Debian GNU/Linux 9 (stretch) | armv5e, armv6, armv7hf, aarch64, amd64, i386 |
+| Debian  | Debian GNU/Linux 10 (Buster) | armv5e, armv6, armv7hf, aarch64, amd64, i386 |
 | Alpine  | Alpine Linux v3.10            | armv6, armv7hf, aarch64, amd64, i386 		|
 | Ubuntu  | 18.04 LTS (Bionic Beaver)    | armv7hf, aarch64, amd64, i386                |
-| Fedora  | Fedora 29 (Twenty Nine)      | armv7hf, aarch64, amd64, i386                |
+| Fedora  | Fedora 30 (Thirty)      | armv7hf, aarch64, amd64, i386                |
 
 | Language | Default  	                  | Supported Architectures                      |
 |---------|------------------------------|----------------------------------------------|
-| Node.js | v12.7.0                      | armv6, armv7hf, aarch64, amd64, i386         |
-| Python  | v2.7.16                       | armv5e, armv6, armv7hf, aarch64, amd64, i386 |
-| OpenJDK | v1.8.0_181                    | armv7hf, aarch64, amd64, i386, armv6         |
-| Go      | 1.12.7                        | armv7hf, aarch64, amd64, i386, armv6         |
-| Dotnet  | 2.2-sdk                       | armv7hf, aarch64, amd64                |
+| Node.js | v13.1.0                      | armv6, armv7hf, aarch64, amd64, i386         |
+| Python  | v2.7.17                       | armv5e, armv6, armv7hf, aarch64, amd64, i386 |
```

Comment:
I wonder if it's also worth having a separate Python3 language tag, bearing in mind the upcoming EOL of Python2 ? https://pythonclock.org/

Reply:
There's also part of the page which says "`latest` points to the latest Python 2 version, which currently is 2.7.17" and I guess that means there's currently no way to automatically get the latest Python 3 version?

Reply:
I missed this one. I'd have thought using `3` would do it but that doesn't seem to work as expected:

```
docker run -it balenalib/raspberrypi4-64-alpine-python:3-3.11-run bash
Python 3.6.9
```

Reply:
Maybe that's the default package available i.e via `apt-get install python3`?

Reply:
Created https://github.com/balena-io-library/base-images/issues/606 to track this inconsistency